api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -28,6 +29,23 @@
    :job_status "succeeded"
    :id job-id})
 
+(use-fixtures :once fixtures/temporary-postgresql-database)
+
+(def ^:private testing-dataset "cd25d59e-1451-44d0-8a24-7669edb9a8f8")
+(def ^:private testing-workspace "wfl-dev/CDC_Viral_Sequencing")
+(def ^:private testing-method-configuration "cdc-covid-surveillance/sarscov2_illumina_full")
+(def ^:private testing-table-name "flowcells")
+(def ^:private testing-column-name "run_date")
+
+(def workload {:id 1})
+
+;; For temporary workspace creation
```

Comment:
`workspace-prefix` and `group` appear to be defined twice and never used.

Reply:
Removed - I think that this was something from a merge conflict that was resolved